kratos/solving_strategies/strategies/butcher_tableau.h

```diff
@@ -66,21 +66,26 @@ namespace Kratos
 template<typename Derived, unsigned int TOrder, unsigned int TSubstepCount>
 class ButcherTableau
 {
+
 public:
+
+    typedef array_1d<double, TSubstepCount> VectorType;
+    typedef array_1d<double,TSubstepCount-1> RowType;
+    typedef std::array<RowType, TSubstepCount-1> MatrixType;
+
     ///@name Type Definitions
     ///@{
 
     static constexpr unsigned int Order() {return TOrder;}
     static constexpr unsigned int SubstepCount() {return TSubstepCount; }
 
-    typedef array_1d<double, SubstepCount()> VectorType;
+    // typedef array_1d<double, SubstepCount()> VectorType;
 
     /* Using the following constructs allows us to multiply parts of vectors with parts of matrices
      * while avoiding BOOST's size checks. This is useful to skip multiplications by zero, since
      * for all explicit runge-kutta methods a_ij = 0 for i>j
      */
```

Comment:
Can we move this comment up top? It was here to justify the odd choice of Matrix type and now it's a bit disconnected.

Reply:
please, modify as required, I was only propposing some changes :)